
From: "Antonino A. Daplas" <adaplas@hotpop.com>

This fixes the logo failing to draw in vga16fb due to faulty boolean logic.

Signed-off-by: Antonino Daplas <adaplas@pol.net>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/video/vga16fb.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN drivers/video/vga16fb.c~fbdev-fix-logo-drawing-failure-for-vga16fb drivers/video/vga16fb.c
--- 25/drivers/video/vga16fb.c~fbdev-fix-logo-drawing-failure-for-vga16fb	Thu Sep  9 16:01:23 2004
+++ 25-akpm/drivers/video/vga16fb.c	Thu Sep  9 16:01:23 2004
@@ -1306,7 +1306,7 @@ void vga16fb_imageblit(struct fb_info *i
 {
 	if (image->depth == 1)
 		vga_imageblit_expand(info, image);
-	else if (image->depth <= info->var.bits_per_pixel)
+	else
 		vga_imageblit_color(info, image);
 }
 
_
